src/services/membrosService.ts

```diff
@@ -1,36 +1,73 @@
 import { MysqlDataSource } from '../config/database';
 import { Membros } from '../entities/membrosEntities';
+import { Admin } from '../entities/adminEntities';
 
 export class MembrosService {
   private membrosRepository = MysqlDataSource.getRepository(Membros);
+  private adminRepository = MysqlDataSource.getRepository(Admin);
 
   private async iniciarDatabase() {
     if (!MysqlDataSource.isInitialized) {
       await MysqlDataSource.initialize();
     }
   }
 
+  async criarMembro(dadosMembro: Partial<Membros>) {
+    await this.iniciarDatabase();
+
+    if (!dadosMembro.adminCriadorId) {
+      throw new Error('Admin Criador não especificado.');
+    }
+
+    const adminCriador = await this.adminRepository.findOne({
+      where: { id: dadosMembro.adminCriadorId }
+    });
+
+    if (!adminCriador) {
+      throw new Error('Admin Criador não encontrado.');
+    }
+
+    const novoMembro = this.membrosRepository.create({
+      ...dadosMembro,
+      adminCriadorId: adminCriador.id
+    });
+
+    return await this.membrosRepository.save(novoMembro);
```

Comment:
remover o await aqui também

`return this.membrosRepository.save(novoMembro);`

Reply:
irei ajustar no próximo PR